src/targets/gpu/kernels/include/migraphx/kernels/dpp.hpp

```diff
@@ -68,11 +68,15 @@ __device__ T dpp_mov(T& x)
     input.data = x;
     for(index_int i = 0; i < n; i++)
     {
+#if defined(__HCC__)
+        output.reg[i] = __llvm_amdgcn_move_dpp(input.reg[i], DppCtrl, RowMask, BankMask, BoundCtrl);
+#else
```

Comment:
I dont think we need to add support for HCC.

Reply:
Why is that ? It would fail to compile with HCC then.

Reply:
Yea it would fail to compile, but hcc was deprecated 3 years ago, so there is no reason to try to support it. We dont even test using hcc either so there is no coverage for this code either.

Reply:
In that case, we also need to clean up HCC from other  places.

Reply:
Removed HCC for now, but later need to clean up other references from entire codebase.